### ISSUE TITLE
Draw circular contigs from GFA assemblies [feature-request]

### ISSUE THREAD

**Issue by govinda-kamath** [rrwick/Bandage#25]
Hi, 

Would it be possible to draw self loops in assemblies? This would help us a lot in bacterial assemblies to distinguish coverage limited partial assemblies from plasmids. 

For example if I have the following GFA

`H	VN:Z:1.0`
`S	1	CGATGCAA`
`L	1 	+ 	1	+	0M`

The visualisation I seem to get is 
<img width="480" alt="screen shot 2016-05-17 at 1 47 59 pm" src="https://cloud.githubusercontent.com/assets/7320464/15338727/0617a594-1c36-11e6-8754-edb1e2241a88.png">

I was wondering if it would be possible to make these into a circular contig in such situations?

Thanks,
Govinda.

**Comment by rrwick** [comment]
Govinda,

Bandage should already behave the way you're requesting. I tried the sample GFA you gave above and got this result in the current Bandage release:
<img width="369" alt="screen shot 2016-07-03 at 9 22 32 am" src="https://cloud.githubusercontent.com/assets/7053555/16542919/b4c0d154-40ff-11e6-8db5-5e77656d490a.png">

The only thing I think of which might explain why it's not working for you is the format of the GFA. When I copied your example, I changed all of the whitespace to tabs, as per the GFA spec:
```
H	VN:Z:1.0
S	1	CGATGCAA
L	1	+	1	+	0M
```
I think GitHub changes the tabs back into spaces in this text, but you get the idea. If your GFA file doesn't use tabs, especially in the link line, Bandage might be failing to parse it correctly and not loading the link.

Ryan
